node-util/sbin/oo-idler-stats

```diff
@@ -131,7 +130,7 @@ def percent(a, b):
 
 
 def cartridge_types(homedir):
-    return_code, types = run("ls |egrep -v '-|git'", homedir)
+    return_code, types = run("ls |egrep -v -e '-|git'", homedir)
```

Comment:
@tiwillia That code didn't work as it failed while I was testing it. Apparently, due to the pattern starting with '-' in '-|git', it's recommended to use -e to avoid the egrep misinterpret the pattern as a option. It's mentioned on manpage.